NeoMathEngine/src/CPU/CpuMathEngine.h

```diff
@@ -36,34 +36,34 @@ class CCpuMathEngine : public IMathEngine, public IRawMemoryManager {
+	void* GetBuffer( const CMemoryHandle& handle, size_t pos, size_t size ) override;
+	void ReleaseBuffer( const CMemoryHandle& handle, void* ptr, bool exchange ) override;
+	void DataExchangeRaw( const CMemoryHandle& handle, const void* data, size_t size ) override;
+	void DataExchangeRaw( void* data, const CMemoryHandle& handle, size_t size ) override;
+	CMemoryHandle CopyFrom( const CMemoryHandle& handle, size_t size ) override;
+	void GetMathEngineInfo( CMathEngineInfo& info ) const override;
 
 	// IVectorMathEngine interface methods
-	virtual void VectorFill(const CFloatHandle& result, float value, int vectorSize) override;
-	virtual void VectorFill(const CIntHandle& result, int value, int vectorSize) override;
-	virtual void VectorFill(const CFloatHandle& result, int vectorSize, const CConstFloatHandle& value) override;
-	virtual void VectorFill(const CIntHandle& result, int vectorSize, const CConstIntHandle& value) override;
-	virtual void VectorFillBernoulli( const CFloatHandle& result, float p, int vectorSize, float value, int seed ) override;
-	virtual void FilterSmallValues( const CFloatHandle& data, int dataSize, float threshold ) override;
-	virtual void VectorCopy(const CFloatHandle& first, const CConstFloatHandle& second, int vectorSize) override;
-	virtual void VectorCopy(const CIntHandle& first, const CConstIntHandle& second, int vectorSize) override;
-	virtual void VectorSum(const CConstFloatHandle& firstHandle, int vectorSize, const CFloatHandle& resultHandle) override;
-	virtual void VectorSumAdd(const CConstFloatHandle& firstHandle, int vectorSize, const CFloatHandle& resultHandle) override;
-	virtual void VectorNegSum(const CConstFloatHandle& firstHandle, int vectorSize, const CFloatHandle& resultHandle) override;
+	void VectorFill(const CFloatHandle& result, float value, int vectorSize) override;
+	void VectorFill(const CIntHandle& result, int value, int vectorSize) override;
+	void VectorFill(const CFloatHandle& result, int vectorSize, const CConstFloatHandle& value) override;
+	void VectorFill(const CIntHandle& result, int vectorSize, const CConstIntHandle& value) override;
+	void VectorFillBernoulli( const CFloatHandle& result, float p, int vectorSize, float value, int seed ) override;
+	void FilterSmallValues( const CFloatHandle& data, int dataSize, float threshold ) override;
+	void VectorCopy(const CFloatHandle& first, const CConstFloatHandle& second, int vectorSize) override;
+	void VectorCopy(const CIntHandle& first, const CConstIntHandle& second, int vectorSize) override;
+	void VectorSum(const CConstFloatHandle& firstHandle, int vectorSize, const CFloatHandle& resultHandle) override;
+	void VectorSumAdd(const CConstFloatHandle& firstHandle, int vectorSize, const CFloatHandle& resultHandle) override;
+	void VectorNegSum(const CConstFloatHandle& firstHandle, int vectorSize, const CFloatHandle& resultHandle) override;
 	virtual void VectorEqual( const CConstIntHandle& firstHandle, const CConstIntHandle& secondHandle,
```

Comment:
In this file there're many multi-line function declarations. I think `virtual` should be removed there too.

But this file doesn't contain interfaces, that's why you can simply add `override` to destructor and remove `virtual` from the whole file.